flutter/shell/platform/tizen/flutter_tizen.cc

```diff
@@ -280,35 +280,33 @@ void FlutterDesktopViewOnKeyEvent(FlutterDesktopViewRef view,
                                   size_t timestamp,
                                   bool is_down) {
 #ifdef NUI_SUPPORT
-  auto* tizen_view = reinterpret_cast<flutter::TizenViewBase*>(
-      ViewFromHandle(view)->tizen_view());
-
-  if (tizen_view->GetType() == flutter::TizenViewType::kView &&
-      ViewFromHandle(view)->engine()->renderer()->type() ==
-          FlutterDesktopRendererType::kEGL) {
-    reinterpret_cast<flutter::TizenViewNui*>(tizen_view)
-        ->OnKey(device_name, device_class, device_subclass, key, string,
-                nullptr, modifiers, scan_code, timestamp, is_down);
+
+  if (auto* tizen_view = dynamic_cast<flutter::TizenViewNui*>(
+          ViewFromHandle(view)->tizen_view())) {
+    if (ViewFromHandle(view)->engine()->renderer()->type() ==
+        FlutterDesktopRendererType::kEGL) {
+      tizen_view->OnKey(device_name, device_class, device_subclass, key, string,
+                        nullptr, modifiers, scan_code, timestamp, is_down);
+    }
   }
+
```

Comment:
No blank lines should be necessary here.

Reply:
oh! thank you.

Reply:
Oh, I mean both 283 and 292nd lines.

Reply:
oh.. thanks again